docs/CONTRIBUTING.md

```diff
@@ -66,9 +66,23 @@ Simply use `exit` to deactivate the environment. The next time you call `poetry
     poetry run mypy .
     ```
 
-4.  Contributions are submitted through [GitHub Pull Requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+4. Test!
 
-    - When you are ready to submit your contribution for review, push your branch:
+    - Run tests to make sure nothing broke
+
+    ```shell
+    poetry run pytest
+    ```
+
+    - Check coverage report
+
+    ```shell
+    poetry run pytest tests/ --cov=./ --cov-report=html
+    ```
+
+5. Contributions are submitted through [GitHub Pull Requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+
+    - When you are  ready to submit your contribution for review, push your branch:
 
     ```shell
     git push origin meaningful-branch-name
```

Comment:
it might also be worth noting below that you can submit the PR by using the Contribute button/feature in the GitHub web page for the repo fork

Reply:
done